src/themes/dark-theme-primitives.js

```diff
@@ -5,59 +5,75 @@ This source code is licensed under the MIT license found in the
 */
 // @flow
+import {colors} from '../tokens/index.js';
 import type {PrimitivesT} from './types.js';
 
 // color constants
 export const primitives: PrimitivesT = {
-  primary50: '#EDF3FE',
-  primary100: '#D2E0FC',
-  primary200: '#9CBCF8',
-  primary300: '#548BF4',
-  primary400: '#276EF1',
-  primary500: '#174EB6',
-  primary600: '#123D90',
-  primary700: '#0C2960',
+  primary: colors.white,
+  primary50: colors.gray50,
+  primary100: colors.gray100,
+  primary200: colors.gray200,
+  primary300: colors.gray300,
+  primary400: colors.gray400,
+  primary500: colors.gray500,
+  primary600: colors.gray600,
+  primary700: colors.gray700,
 
-  negative50: '#FDF0EF',
-  negative100: '#FADBD7',
-  negative200: '#F4AFA7',
-  negative300: '#EB7567',
-  negative400: '#E54937',
-  negative500: '#AE372A',
-  negative600: '#892C21',
-  negative700: '#5C1D16',
+  accent: colors.blue400,
+  accent50: colors.blue50,
+  accent100: colors.blue100,
+  accent200: colors.blue200,
+  accent300: colors.blue300,
+  accent400: colors.blue400,
+  accent500: colors.blue500,
+  accent600: colors.blue600,
+  accent700: colors.blue700,
 
-  warning50: '#FEF3EC',
-  warning100: '#FBE2CF',
-  warning200: '#F6BA8B',
-  warning300: '#F19248',
-  warning400: '#ED6F0E',
-  warning500: '#B4540B',
-  warning600: '#8E4308',
-  warning700: '#5F2C06',
+  negative: colors.red400,
+  negative50: colors.red50,
+  negative100: colors.red100,
+  negative200: colors.red200,
+  negative300: colors.red300,
+  negative400: colors.red400,
+  negative500: colors.red500,
+  negative600: colors.red600,
+  negative700: colors.red700,
 
-  positive50: '#EBF8F2',
-  positive100: '#CDEDDE',
-  positive200: '#88D3B0',
-  positive300: '#43B982',
-  positive400: '#07A35A',
-  positive500: '#057C44',
-  positive600: '#046236',
-  positive700: '#034124',
+  warning: colors.orange400,
+  warning50: colors.orange50,
+  warning100: colors.orange100,
+  warning200: colors.orange200,
+  warning300: colors.orange300,
+  warning400: colors.orange400,
+  warning500: colors.orange500,
+  warning600: colors.orange600,
+  warning700: colors.orange700,
 
-  mono100: '#CCCCCC',
-  mono200: '#999999',
-  mono300: '#7A7A7A',
-  mono400: '#5C5C5C',
-  mono500: '#3D3D3D',
+  positive: colors.green400,
+  positive50: colors.green50,
+  positive100: colors.green100,
+  positive200: colors.green200,
+  positive300: colors.green300,
+  positive400: colors.green400,
+  positive500: colors.green500,
+  positive600: colors.green600,
+  positive700: colors.green700,
+
+  mono100: colors.gray300,
+  mono200: colors.gray400,
+  mono300: colors.gray500,
+  mono400: colors.gray600,
+  mono500: colors.gray700,
+  // mono600-900 are not in official brand tokens atm
   mono600: '#292929',
   mono700: '#1F1F1F',
   mono800: '#141414',
   mono900: '#111111',
-  mono1000: '#000000',
+  mono1000: colors.black,
 
-  rating200: '#FFE1A5',
-  rating400: '#FFC043',
+  rating200: colors.yellow200,
+  rating400: colors.yellow400,
```

Comment:
Since we already adding breaking changes to the theme, maybe we should get rid of (remap to semantically named color scale) the `ratingX00` colors and add something like `accentSecondary`.